Formula/cpmtools.rb

```diff
@@ -11,8 +11,18 @@ class Cpmtools < Formula
     sha256 "b810122c220af6b36ab9316deec811adca68313d4371f8a0121239c40b94a015" => :mavericks
   end
 
+  option "with-libdsk", "Enable libdsk support"
+
+  depends_on "libdsk" => :build if build.with? "libdsk"
```

Comment:
Are we sure this is just build? Haven't checked & don't know, but seems odd to only be `:build`.